From cdb4bef7a695401b4dcaee43c70635960f60d697 Mon Sep 17 00:00:00 2001
From: Andreas Blaesius <skate4life@gmx.de>
Date: Sat, 4 Nov 2017 15:54:54 +0100
Subject: [PATCH 4/5] Remove QuickSearchBox from build

outdated...tons of aapt warnings and no one is using it:
remove it!

Change-Id: If1c83e9bca9e91ae3cd47df3414c42b323890a54
---
 configs/common.mk          |  4 ++++
 remove_packages/Android.mk | 11 +++++++++++
 2 files changed, 15 insertions(+)
 create mode 100644 remove_packages/Android.mk

diff --git a/configs/common.mk b/configs/common.mk
index 1ffedff..1ee7967 100644
--- a/configs/common.mk
+++ b/configs/common.mk
@@ -43,3 +43,7 @@ PRODUCT_PACKAGES += \
     Launcher3 \
     Terminal \
     WallpaperPicker
+
+# Remove packages unsafe or unused default packages
+PRODUCT_PACKAGES += \
+    RemovePackages
diff --git a/remove_packages/Android.mk b/remove_packages/Android.mk
new file mode 100644
index 0000000..de49ee4
--- /dev/null
+++ b/remove_packages/Android.mk
@@ -0,0 +1,11 @@
+LOCAL_PATH := $(call my-dir)
+include $(CLEAR_VARS)
+
+LOCAL_MODULE := RemovePackages
+LOCAL_MODULE_TAGS := optional
+LOCAL_MODULE_CLASS := APPS
+LOCAL_OVERRIDES_PACKAGES := QuickSearchBox
+LOCAL_UNINSTALLABLE_MODULE := true
+LOCAL_CERTIFICATE := PRESIGNED
+
+include $(BUILD_PREBUILT)
-- 
2.7.4

